templates/admin/auth/edit.tmpl

```diff
@@ -211,8 +211,8 @@
 					</div>
 					<div class="has-tls inline field {{if not .HasTLS}}hide{{end}}">
 						<div class="ui checkbox">
-							<label><strong>{{.locale.Tr "admin.auths.skip_tls_verify"}}</strong></label>
-							<input name="skip_verify" type="checkbox" {{if .Source.SkipVerify}}checked{{end}}>
+							<label for="skip_verify"><strong>{{.locale.Tr "admin.auths.skip_tls_verify"}}</strong></label>
+							<input id="skip_verify" name="skip_verify" type="checkbox" {{if .Source.SkipVerify}}checked{{end}}>
```

Comment:
There is a `skip_verfiy` further up

Reply:
That lines are from another Auth source case. It will never be rendered at same time.